src/components/Card/Card.tsx

```diff
@@ -6,6 +6,7 @@ export enum CardRatioOptions {
   RATIO_50_50 = '50',
   RATIO_40_60 = '40',
   RATIO_30_70 = '30',
```

Comment:
let's add examples to each one of the aspect ratio.
they can all be in the same code box